test/unit/tasks/migrate_role_content_test.rb

```diff
@@ -0,0 +1,26 @@
+require "test_helper"
+require "rake"
+class MigrateRoleContentTest < ActiveSupport::TestCase
```

Comment:
Nice! We don't often unit test Rake tasks, this is pretty nice to see.

Reply:
I spent half of Friday trying to manually test it and then realized with a unit test I can just use the factories to do it instead. 🙂